doc/examples/uncertainty_emcee.ipynb

```diff
@@ -1,205 +1,324 @@
+    "\n",
+    "In this notebook it is shown how the MCMC-algorithm can be used to estimate the model parameters and quantify the (parameter) uncertainties for a Pastas model using a Bayesian approach. For this the EmceeSolver is introduced, based on the [emcee](https://emcee.readthedocs.io) Python package. \n",
+    "\n",
+    "Besides Pastas the following Python Packages have to be installed to run this notebook:\n",
+    "\n",
+    "- [emcee](https://emcee.readthedocs.io)\n",
+    "- [corner](https://corner.readthedocs.io)\n",
+    "\n",
+    "<div class=\"alert alert-warning\">\n",
+    "<b>Note:</b>\n",
+    "The EmceeSolver is still an experimental feature and some of the arguments might be changed in the near future (2023/06/22). We welcome testing and feedback on this new feature!.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "import pastas as ps\n",
+    "import emcee\n",
+    "import corner\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "ps.set_log_level(\"ERROR\")\n",
+    "ps.show_versions()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Create a Pastas Model\n",
+    "The first step is to create a Pastas Model, including the RechargeModel to simulate the effect of precipitation and evaporation on the heads. Here, we first estimate the model parameters using the standard least-squares approach."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "head = pd.read_csv(\n",
+    "    \"data/B32C0639001.csv\", parse_dates=[\"date\"], index_col=\"date\"\n",
+    ").squeeze()\n",
+    "\n",
+    "# Make this millimeters per day\n",
+    "evap = pd.read_csv(\"data/evap_260.csv\", index_col=0, parse_dates=[0]).squeeze()\n",
+    "rain = pd.read_csv(\"data/rain_260.csv\", index_col=0, parse_dates=[0]).squeeze()\n",
+    "\n",
+    "ml = ps.Model(head)\n",
+    "\n",
+    "# Select a recharge model\n",
+    "rch = ps.rch.FlexModel()\n",
+    "\n",
+    "rm = ps.RechargeModel(rain, evap, recharge=rch, rfunc=ps.Gamma(), name=\"rch\")\n",
+    "ml.add_stressmodel(rm)\n",
+    "\n",
+    "ml.solve(noise=True, tmin=\"1990\")\n",
+    "\n",
+    "ax = ml.plot(figsize=(10, 3))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Use the EmceeSolver\n",
+    "\n",
+    "We will now use the EmceeSolve solver to estimate the model parameters and their uncertainties. This solver wraps the [Emcee](https://lmfit.github.io/lmfit-py/fitting.html#lmfit.minimizer.Minimizer.emcee) package, which implement different versions of MCMC. A good understanding of Emcee helps when using this solver, so it comes recommended to check out their documentation as well.\n",
```

Comment:
Perhaps mention something about how the estimated stderrs from the previous solve are used to define the priors?